css/style.css

```diff
@@ -0,0 +1,477 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:400,700');
+body{
```

Comment:
what is the use of `style2.css` file. if not needed please remove it

Reply:
Have removed